shellweb

```diff
@@ -290,6 +294,24 @@ EOF
 	printf "%s" "$body"
 }
 
+proxy() {
+	uri_port=${uri##*:}
```

Comment:
If `uri` be really a URI, this would be wrong, as it'll convert, e.g., `http://example.com` to `http`. ;) But given that `$uri` is actually a TCP/IP address specification, this is okay.